tests/__init__.py

```diff
@@ -1 +1,2 @@
-# placeholder file to modularize (to package) .py files under the directory "tests", making the files importable.
\ No newline at end of file
+# placeholder file to modularize (to package) .py files
+# under the directory "tests", making the files importable.
```

Comment:
See comment about extending line length and revert this.

Reply:
Done.